src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
+function numberWithCommas(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
+
 // write your code here
+function main() {
+  const doc = document.documentElement;
+  let popul = [...doc.getElementsByClassName('population')];
+
+  popul = popul.map((elem) => elem.innerHTML.split(',').join(''));
+
+  const total = popul.reduce((prev, elem) => +elem + prev, 0);
+  const average = Math.floor(total / popul.length);
+
+  const totalPopulEl = doc.getElementsByClassName('total-population')[0];
+  const avgPopulEl = doc.getElementsByClassName('average-population')[0];
```

Comment:
Similarly, ensure that the element with class 'average-population' exists in the DOM. Adding a check to handle cases where the element might not be found would be beneficial to avoid runtime errors.